llvm/lib/CodeGen/WindowScheduler.cpp

```diff
@@ -451,7 +451,6 @@ int WindowScheduler::calculateMaxCycle(ScheduleDAGInstrs &DAG,
       }
       RM.reserveResources(*SU, CurCycle);
     }
-    RM.reserveResources(*SU, CurCycle);
```

Comment:
This line is really redundant. It will cause the instruction to occupy too many resources and fail to emit properly.